### ISSUE TITLE
/articles mobile: can't tap on inline link if it's also a glossary term

### ISSUE THREAD

**Issue by ricewithabee** [StampyAI/stampy-ui#534]
![Image](https://github.com/StampyAI/stampy-ui/assets/98026499/262ff3c5-76b9-4810-84f6-964ea8a1ef96)

My best guess at a solution is to make it so that glossary definitions aren't attached to text which contains a link.

**Comment by melissasamworth** [comment]
Oh gosh that's tricky. I can't think of a better solution than yours at the moment.

**Comment by mruwnik** [comment]
would this only be on mobile, or in general?

**Comment by mruwnik** [comment]
how about long tap?

**Comment by bryceerobertson** [comment]
@mruwnik this is only a problem on mobile – because you can't hover on mobile

**Comment by bryceerobertson** [comment]
@mruwnik is long tap on websites a thing??

**Comment by mruwnik** [comment]
it could be - I vaguely recall that being supported, and if not it shouldn't be that hard to implement it. The main issue would be communicating that functionality to the user

**Comment by bryceerobertson** [comment]
Yeah agreed. @melissasamworth any thoughts on how to communicate this to users?

**Comment by LeMurphant** [comment]
I don't think it matters much on desktop. Also, the content team has been attempted to find instances of this link + glossary and remove the link, so it should become less of a problem over time, but I still think that it should be fixed with a technical solution as well.

**Comment by bryceerobertson** [comment]
@LeMurphant nice. Given that + since there doesn't seem to be any great technical solution I think this issue is low priority then

**Comment by LeMurphant** [comment]
Will be fixed by https://github.com/StampyAI/stampy-ui/issues/534

**Comment by bryceerobertson** [comment]
@LeMurphant I'm stuck in an infinite loop (this is #534)

**Comment by LeMurphant** [comment]
Apologies @bryceerobertson , that was a bad copy paste, I meant https://github.com/StampyAI/stampy-ui/pull/917, which is now merged. Feel free to re-open this issue if it does not work as expected.

**Comment by bryceerobertson** [comment]
No worries, looks good
